src/app/services/auth.service.ts

```diff
@@ -84,6 +84,10 @@ export class AuthService {
       );
   }
 
+  userId(): string | undefined {
```

Comment:
return string | null (null if currentUser or currentUser?._id is undefined) please